src/workerd/api/actor-state.h

```diff
@@ -470,6 +477,8 @@ class DurableObjectState: public jsg::Object {
     JSG_METHOD(setWebSocketAutoResponse);
     JSG_METHOD(getWebSocketAutoResponse);
     JSG_METHOD(getWebSocketAutoResponseTimestamp);
+    JSG_METHOD(setHibernatableWebSocketEventTimeout);
+    JSG_METHOD(getHibernatableWebSocketEventTimeout);
```

Comment:
Non-blocking: would prefer these as a property getter/setter pair but this is fine as is.

Reply:
I could do that, but this way is more consistent with how we do for other hibernatable websocket methods.